src/lib/io/sol-fs-storage.c

```diff
@@ -70,11 +73,14 @@ fs_write(const char *name, struct sol_buffer *buffer)
     return ret;
 }
 
-int
-fs_read(const char *name, struct sol_buffer *buffer)
+SOL_API int
+sol_fs_read_raw(const char *name, struct sol_buffer *buffer)
```

Comment:
ouch, I did read the file name `src/modules/flow/persistence/fs-storage.c`, didn't notice it was moved to `src/lib/io/sol-fs-storage.c`. my bad